astropy/coordinates/tests/test_representation_methods.py

```diff
@@ -367,14 +367,14 @@ def test_atleast_3d(self):
 
     def test_move_axis(self):
         # Goes via transpose so works without __array_function__ as well.
-        s0_10 = np.moveaxis(self.s0, 0, 1)
+        s0_10 = np.moveaxis(self.s0.data, 1, 0)
         assert s0_10.shape == (self.s0.shape[1], self.s0.shape[0])
         assert np.all(representation_equal(self.s0.T, s0_10))
         assert np.may_share_memory(s0_10.lon, self.s0.lon)
 
     def test_roll_axis(self):
         # Goes via transpose so works without __array_function__ as well.
-        s0_10 = np.rollaxis(self.s0, 1)
+        s0_10 = np.moveaxis(self.s0, 1)
```

Comment:
```suggestion
        s0_10 = np.moveaxis(self.s0, 1, 0)
```
`destination` is a required argument to `moveaxis`. I _think_ this should be the correct setting for the default `start=0`.